packages/plugins/SmartPay/src/hooks/useDeploy.tsx

```diff
@@ -43,6 +42,7 @@ export function useDeploy(
     )
     const connection = useWeb3Connection(NetworkPluginID.PLUGIN_EVM, {
         providerType: ProviderType.MaskWallet,
+        account: undefined,
```

Comment:
What's the difference?

Reply:
Avoid being contaminated by account on the chain context.